horace_core/sqw/file_io/@dnd_binfile_common/dnd_binfile_common.m

```diff
@@ -134,7 +134,7 @@
                 error('SQW_FILE_IO:runtime_error',...
                     'dnd_binfile_common:init_from_sqw_obj method should be invoked with at least an existing sqw or dnd object provided');
             end
-            if isa(varargin{1},'sqw_old') || is_sqw_struct(varargin{1})
+            if isa(varargin{1},'sqw') || is_sqw_struct(varargin{1})
```

Comment:
do these checks need to be `sqw_old` || `sqw`?

Reply:
We have basically moved a lot of the code so it is using sqw rather than sqw_old. The more sensible thing at this point would be to detect sqw_old as an error condition. But as this change was made following a test failure, this functionality already tests for sqw_old as a fail state, and it should be enough just to leave this code as-is.